templates/players/subscribe-links.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Can we make this regular HTML, with PHP echos inside the HTML, instead of the other way around. Thanks